packages/utils/src/math/range/index.ts

```diff
@@ -0,0 +1,34 @@
+import { isNil } from '../../validator';
+
+const getRangeValue = (start: number, end: number, step: number = 1) => {
+  const result = [];
+
+  for (let value = start; value <= end; value += step) {
+    result.push(value);
+  }
+
+  return result;
+};
+
+export function range(end: number): number[];
+export function range(start: number, end: number): number[];
+export function range(start: number, end: number, step?: number): number[];
+export function range(startOrEnd: number, end?: number, step?: number) {
```

Comment:
```suggestion
export function range(start: number, end?: number, step?: number) {
```
함수 오버로딩 네이밍을 맞추는게 적절해보입니다 :)